classes/class-kco-gateway.php

```diff
@@ -396,6 +396,37 @@ private function check_if_eu( $store_base_location ) {
 			}
 		}
 
+		/**
+	 	 * Get required setting keys for setup.
+		 *
+		 * @return array Array of setting keys used for setup.
+		 */
+		public function get_required_settings_keys() {
+			$setting_suffix = $this->check_if_eu( $store_base_location['country'] ) === 'EU' ? 'eu' : 'us';
```

Comment:
`$store_base_location` is undefined here which I think is the root cause of [this comment](https://github.com/woocommerce/woocommerce-admin/pull/7085#pullrequestreview-674370705).

```suggestion
			$store_base_location = wc_get_base_location();
			$setting_suffix = $this->check_if_eu( $store_base_location['country'] ) === 'EU' ? 'eu' : 'us';
```

Reply:
Ah thanks so much for spotting this @joshuatf !